CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 # History
 
+## 3.2.0
```

Comment:
```suggestion
## Unreleased
```

?

Reply:
It's up to you, I know nothing about your agreements in versioning and all